libcontainer/cgroups/fs/error.go

```diff
@@ -0,0 +1,15 @@
+package fs
+
+import (
+	"fmt"
+
+	"github.com/opencontainers/runc/libcontainer/cgroups/fscommon"
+)
+
+type ParseError = fscommon.ParseError
+
+// malformedLine is used by all cgroupfs file parsers that expect a line
+// in a particular format but get some garbage instead.
+func malformedLine(path, file, line string) error {
```

Comment:
Shouldn't this be in fscommon?

Reply:
For now this is specific to cgroups/fs only -- no cgroups/fs2 parses returns an error like this. For some reason fs2 parsers all more forgiving -- they skip, rather than error on, unknown lines.

If there will be a need we can move this to fscommon later.